VSOWorkBot/Extensions/AuthHelper.cs

```diff
@@ -96,6 +96,11 @@ public async Task<string> GetTokenAsync(string conversationId, string userId)
     {
         // Query all entities in the table
         var activityEntities = await table.ExecuteAsync(TableOperation.Retrieve<BotTokenEntity>(conversationId, userId, botTokenEntityResolver));
+        if (activityEntities?.Result == null || !(activityEntities.Result is BotTokenEntity))
```

Comment:
We should catch instead and handle the error instead of failing silently.

Reply:
Will do it later